eclipse/src/saros/ui/wizards/pages/ResourceSelectionWizardPage.java

```diff
@@ -130,7 +133,13 @@ public void run() {
              * properly.
              */
             resourceSelectionComposite.rememberSelection();
-            setPageComplete(selectedResources.size() > 0);
+
+            if (selectedResources.size() > 0) {
+              setPageComplete(true);
+            } else {
+              setErrorMessage(Messages.ProjectSelectionWizardPage_selected_no_project);
```

Comment:
So the wizard shows always an error message when it is displayed ?

Reply:
If there is no default selection, yes. This error message is displayed as part of the wizard itself. It gives the user information why the dialog can not be continued.

Reply:
This is an UX issue. You simply do not start a wizard with displaying an error. The normal case would be to display this error after a user selected some resource and finally deselect them all. When the wizard opens there should be an information how the wizard should be used, e.g what the user should click / entering in order to become the Finish button to be enabled.

Reply:
Done. Just set the wizard as non-completable if there is no default selection.